ROMFS/px4fmu_common/init.d/airframes/4401_ssrc_fog_x_tmotor

```diff
@@ -79,7 +79,7 @@ param set-default BAT2_CAPACITY 22000
 param set-default SENS_EN_LL40LS 2
 
 # Disable internal magnetometer
-param set CAL_MAG0_PRIO 0
```

Comment:
The comment says "Disable internal magnetometer". If you want to keep it enabled (but set low priority), maybe change the comment above as well, thanks :)

Reply:
good catch, should be fixed now